readme.md

```diff
@@ -50,7 +50,7 @@ This package improves [`child_process`](https://nodejs.org/api/child_process.htm
 - Redirect [`stdin`](#stdin)/[`stdout`](#stdout-1)/[`stderr`](#stderr-1) from/to files, streams, iterables, strings, `Uint8Array` or [objects](docs/transform.md#object-mode).
 - [Transform](docs/transform.md) `stdin`/`stdout`/`stderr` with simple functions.
 - Iterate over [each text line](docs/transform.md#binary-data) output by the process.
-- [Graceful termination](#optionsforcekillaftertimeout).
+- [Graceful termination](#forcekillaftertimeout).
```

Comment:
We should use a better word than "graceful" as it can be misunderstood.

Reply:
Done in 731208291c2d153ba51fe4f1bfe0ee2ced8bbc4b